src/terrain/routes/metadata.clj

```diff
@@ -17,6 +17,7 @@
         [common-swagger-api.schema.apps.rating]
         [common-swagger-api.schema.integration-data :only [IntegrationData]]
         [ring.util.http-response :only [ok]]
+        [terrain.auth.user-attributes]
```

Comment:
Small, but unless there's a reason you need to include the whole namespace, this should probably be:

```suggestion
        [terrain.auth.user-attributes :only [require-authentication]]
```

Most clojure lint tools complain about "unlimited `use`" since without being explicit about it, it's relatively easy to end up with weird stuff in your namespace, and it's more opaque.